<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtToolEncrangerMainDialog</class>
 <widget class="QDialog" name="QtToolEncrangerMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Encranger</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ToolEncranger.qrc">
    <normaloff>:/images/ToolEncrangerArrowDown16x16.png</normaloff>:/images/ToolEncrangerArrowDown16x16.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="5">
    <widget class="QLabel" name="label_arrow_up">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="ToolEncranger.qrc">:/images/ToolEncrangerArrowUp34x34.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="button_deencrypt">
     <property name="text">
      <string>Deencrypt</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="box_key">
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>12345</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="button_encrypt">
     <property name="text">
      <string>Encrypt</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_arrow_down">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="ToolEncranger.qrc">:/images/ToolEncrangerArrowDown34x34.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QLineEdit" name="edit_plaintext">
     <property name="text">
      <string>This is plaintext</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_key">
     <property name="text">
      <string>Key</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QLineEdit" name="edit_encrypted_text"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ToolEncranger.qrc"/>
 </resources>
 <connections/>
</ui>
